### ISSUE TITLE
Improve usability of details panel indicators

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4557]
**What we're after:**
Improve the appearance of the details panel buttons/indicators. At present, users can click anywhere on the row to get to additional information nested in the details panel. But there's also the indictor at the far right side of the row that suggests there's additional content. In gray, it's not entirely clear that there's additional information and not everyone is savvy enough to pat attention to the changing cursor when looking at the rows of content. 

**Completion criteria:**
- [ ] Improve appearance of data table side panel indicators to improve navigation to additional report/transaction details

Current state:
![Screen Shot 2021-04-19 at 10.52.55 AM.png](https://images.zenhubusercontent.com/59b1a5f3b0222d5de476d238/fc2e8578-abd6-4eb1-9f05-d5032e6d63f8)

**Comment by patphongs** [comment]
Thanks @JonellaCulmer, I do like the new panel indicators. I have a couple of questions about functionality/display below:

1. Clicking on each plus sign would push table rows down. How many can be opened at once? It may be disorienting to allow them to open more than one at a time?
2. In the current implementation, the entire row is clickable to trigger the opening of the panel. Will the + and - function in the same way? If the entire row is clicked, will it open and if the row is clicked again, will it close?
3. It'll be interesting to see how the new horizontal panel will stack in full width view. Especially when there are 3 or more blocks/sections to display for one item.

**Comment by JonellaCulmer** [comment]
@patphongs 

> Clicking on each plus sign would push table rows down. How many can be opened at once? It may be disorienting to allow them to open more than one at a time?

I agree. We should maintain the functionality now that allows only one panel to be open at a time. 

> In the current implementation, the entire row is clickable to trigger the opening of the panel. Will the + and - function in the same way? If the entire row is clicked, will it open and if the row is clicked again, will it close?

I'd like that to be the case, yes. That clicking anywhere on the row can either open or close it. It would be new functionality to click anywhere on the row to close. Currently, we use the X to close. 

> It'll be interesting to see how the new horizontal panel will stack in full width view. Especially when there are 3 or more blocks/sections to display for one item.

What I imagine in my mind does not always translate 1:1 on the front-end so I need to think through this a bit more. Perhaps this part of the implementation can be a pairing opportunity in addition to some more thought on my end.
